sdk/formrecognizer/Azure.AI.FormRecognizer/src/CustomFormModel.cs

```diff
@@ -112,42 +136,89 @@ private static IReadOnlyList<CustomFormSubmodel> ConvertFromUnlabeled(KeysResult
                 subModels.Add(new CustomFormSubmodel(
                     $"form-{cluster.Key}",
                     default,
-                    fieldMap));
+                    fieldMap,
+                    model.ModelInfo.ModelId));
             }
             return subModels;
         }
 
         private static IReadOnlyList<CustomFormSubmodel> ConvertFromLabeled(Model model)
+        {
+            return new List<CustomFormSubmodel> {
+                new CustomFormSubmodel(
+                    $"form-{model.ModelInfo.ModelId}",
```

Comment:
It occurred to me that we need to update the `form_type` on CustomFormSubmodel. This should match what is returned in the `RecognizedForm`. This changed to `$"custom-{model.ModelInfo.ModelId}"` in v2.1

Reply:
Great point. I will update

Reply:
After looking at the service response for `RecognizedForm` I am assigning `formType` according to the `displayName` value.